products/sle/main.pm

```diff
@@ -801,8 +801,13 @@ elsif (get_var("VIRT_AUTOTEST")) {
     if (get_var("VIRT_PRJ1_GUEST_INSTALL")) {
         loadtest "virt_autotest/guest_installation_run";
         if (!(get_var("GUEST_PATTERN") =~ /win/img) && is_x86_64) {
-            loadtest "virt_autotest/setup_dns_service";
             loadtest "virt_autotest/set_config_as_glue";
+            loadtest "virt_autotest/libvirt_virtual_network_init"        if get_var("ENABLE_VIR_NET");
```

Comment:
why not in one if block?

Reply:
Great, had made vir nerwork tests in one if block.